pipeline/stage02_processing/Snakefile

```diff
@@ -164,9 +164,7 @@ rule spatial_downsampling:
         data = os.path.join('{dir}', 'spatial_downsampling',
                             '{rule_name}.'+NEO_FORMAT),
```

Comment:
It doesn't really change anything but just for consistency in the rule definitions, it should rather be `'{rule_name}', 'spatial_downsampling'`